website/docs/how-to/how-to-run-the-unleash-proxy.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: How to run the Unleash Proxy
+---
+import ApiRequest from '@site/src/components/ApiRequest'
+
+:::info Placeholders
+Placeholders in the code samples below are delimited by angle brackets (i.e. `<placeholder-name>`). You will need to replace them with the values that are correct in _your_ situation for the code samples to run properly.
+:::
+
+
+The [Unleash Proxy](../sdks/unleash-proxy.md) provides a way for you to consume feature toggles in [front-end clients](../sdks/index.md#front-end-sdks), such as the [JavaScript Proxy client](../sdks/proxy-javascript.md) and [React Proxy client](../sdks/proxy-react.md).
+
+Depending on your setup, the Proxy is most easily run in one of two ways, depending on your situation:
+- [Via Docker](#run-proxy-via-docker)
+- [As a Node.js app](#run-proxy-as-node-app)
+
+If you're using a hosted version of Unleash, we can also run the proxy for you.
+
+## Prerequisites
+
+This is what you need before you can run the proxy:
+
+- A running Unleash server to connect to. You'll need its API path (e.g. `https://app.unleash-hosted.com/demo/api`) to connect the proxy to it.
+- A [client API token](../reference/api-tokens-and-client-keys#client-tokens) for the proxy to use.
```

Comment:
What's our style around ending list items with fullstops again? This list ends with fullstops, the one above doesn't have any. I promise I will write this down somewhere this time so I stop asking this question

Reply:
I don't think you've asked that before, but it's a good question. The way I've tried to do it so far is:

- if the bullet points are single sentences, don't end in full stops
- if the bullet points _do_ consist of multiple sentences, end them in full stops (so that you don't have an awkward trailing, stopless sentence)
- if _one_ bullet ends in a full stop, _all_ bullets should

But it might also depend on the content of the bullet points, etc. Sometimes it would feel inappropriate to end in full stops, such as:

> You can use this for:
> - Cooking.
> - Cleaning.
> - And much more.

 I'm open to hearing suggestions, though!

Reply:
A suggestion might be:
- If the bullet points contain sentences, end them in full stops.
- If the bullet points are single words or similar, don't use full stops.
- Each list should be consistent with itself.

Does that make sense?

Reply:
This makes perfect sense! I'm going to write this down somewhere and start a style guide